src/Discord.Net.WebSocket/DiscordSocketApiClient.cs

```diff
@@ -202,8 +230,7 @@ public Task SendGatewayAsync(GatewayOpCode opCode, object payload, RequestOption
             {
                 Token = AuthToken,
                 Properties = props,
-                LargeThreshold = largeThreshold,
-                UseCompression = useCompression,
+                LargeThreshold = largeThreshold
```

Comment:
Trailing comma?

Reply:
I removed it since it didn't make sense. It doesn't harm functionality.

Reply:
The reason it's typically included is for Diffs, so it shows as a single-line change vs two.

Reply:
I'm fairly sure we don't do it in other places... There's no point changing it back now because it'll show both commits.